src/Core/Commander/Scheduler.js

```diff
@@ -120,7 +120,10 @@ Scheduler.prototype.execute = function execute(command) {
     const q = host ? this.hostQueues.get(host) : this.defaultQueue;
 
     // execute command now if possible
-    if (q.counters.executing < this.maxCommandsPerHost) {
+    if (forceRunNow) {
```

Comment:
Not sure about this. Wouldn't it be simpler to simply call `provider.executeCommand` if `searchInParent` is true in NodeProcess.processNodeImagery or Elevation?